tests/rados/stretch_cluster.py

```diff
@@ -191,6 +239,30 @@ def run(ceph_cluster, **kw):
     return 0
 
 
+def change_recover_threads(node: CephAdmin, config: dict, action: str):
+    """
+    increases or decreases the recovery threads based on the action sent
+    Args:
+        node: Cephadm node where the commands need to be executed
+        config: Config from the suite file for the run
+        action: Set or remove increase the backfill / recovery threads
+            Values : "set" -> set the threads to specified value
```

Comment:
Using Exact strings that are used by command.  I am just passing what strings command use, so that i avoid mapping of values sent to be used in command.

Reply:
Thanks for clarifying, It is good to have 1 to 1 mapping.